### DISCUSSION TITLE
Systematic Literature Review Tooling

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#31] [General]
This is not for this MSc but for your notes. 
Might be handy in the future or handy to close some open questions.

There are two very nice tools that can help filter through a large set of documents (references)
the Application of these tools are not an easy thing to do and a research topic in itself.

By AI/ML: AsReview
https://asreview.readthedocs.io/en/latest/intro/about.html#the-case-of-systematic-reviewing

by R and data science
Bibliometrix + Biblioshiny
https://www.bibliometrix.org/Biblioshiny.html

More on this I will keep documented on 
https://gitlab.com/edzob/complex_adaptive_systems-knowledge_base/-/wikis/Snowballing-Tooling

**Comment by JRBliekendaal** [comment]
if I had this a year ago I probably would have tried to use them ;) But for now it is too late 👯 Nice for in the archive off things.
